modules/gamma/main.nf

```diff
@@ -0,0 +1,40 @@
+def VERSION = '2.1' // Version information not provided by tool on CLI
+
+process GAMMA {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::gamma=2.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gamma%3A2.1--hdfd78af_0':
+        'quay.io/biocontainers/gamma:2.1--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    path(db)
+
+    output:
```

Comment:
Missing `--fasta`

```
-f, --fasta: Writes out a multifasta file of the gene matches
```

Reply:
This is an optional argument that can be included as an "args"

Reply:
Yes sorry I wasn't clear: I meant more that the module would not pick up the output file that comes from that flag.

You don't have to add a varible _within_ the module to execute it, but you must make sure that file does get publishd if a user DOES specify `-f` via $args.

Reply:
Thank you for clarifying @jfy133 
I have added it to the output and also ran a test to confirm.